src/components/views/rooms/Stickerpicker.js

```diff
@@ -35,6 +35,7 @@ const STICKERPICKER_Z_INDEX = 3500;
 // Key to store the widget's AppTile under in PersistedElement
 const PERSISTED_ELEMENT_KEY = "stickerPicker";
 
+// TODO figure out where to expose it now that the EmojiPicker has taken its place
```

Comment:
Hmm, so is it just going to be inaccessible for a while?

Reply:
this comment is just stale, we now have a button for each